benchmarks/local-send-recv.py

```diff
@@ -31,20 +31,36 @@
 mp = mp.get_context("spawn")
 
 
+def register_am_allocators(args):
+    if not args.enable_am:
+        return
```

Comment:
If the use tried to call this, should we instead return False if `enable_am` is False as a sanity check?

Reply:
Sorry, this in the test not the ucp package, you can skip over my previous comment